include/sort.h

```diff
@@ -2,7 +2,7 @@
 /*! \file sort.h
 ** \verbatim
 ** Top contributors (to current version):
-**   Makai Mann, Clark Barrett
+**   Lindsey Stuntz
```

Comment:
Not important, but I think this one should stay the same since there are no changes to the file. It's good to have it be correct so people know who to contact.